1_OCaml/TP/TP2.ipynb

```diff
@@ -31,6 +31,33 @@
     "6. Soient $a$ et $b$ deux entiers et $q, r$ le quotient et reste de la division euclidienne de $a$ par $b$ ($a = bq + r$). En utilisant le fait que $PGCD(a, b)$ = $PGCD(b, r)$, écrire une fonction `euclide` telle que `euclide a b` renvoie le PGCD de $a$ et $b$."
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        else if n mod 2 =0 then let b = (exp_rapide a (n/2)) in (exp_rapide b 2)
</pre><p>Intéressant</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>